.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+<!--
```

Comment:
Note for @jai to suggest a new PR template with checklists to make sure that people:

* Have typed a description
* Have used a proper PR type
* etc